canonical-web-and-design/ubuntu.com
# Add the smart-start guide as static content

## Done
- Add the smart-start guide as static content from discourse.
- Created the get-in-touch form.
- Add figcaptions to all images in the smart-start section
- Added Vanilla pagination to the bottom of each guide
- Pushed all images to the asset server
- Added copydoc links to each page
- Used Vanilla stepped list when appropriate 
- Updated the navigation to match the page nav

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/smart-start/guide
- Check the content matches the content on live
- Check the navigation matches [the copy doc](https://docs.google.com/document/d/1o3IUaCCPcog6fBxQFYkUAFdFXsceM_6qoyjHWNn727w/edit) at the top
- Go to #get-in-touch on any page and check the dynamic form works and matches [the copydoc](https://docs.google.com/document/d/1DobeTFjupLbGl4TUMu4f4pPAqNMYrR6U4U303ioYkV4/edit#)
- Check all the pagination on pages work as expected

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/8451
Fixes #8433
